src/Razor/test/Microsoft.VisualStudio.LanguageServer.ContainedLanguage.Test/UriExtensionsTest.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Razor;
+using Microsoft.VisualStudio.LanguageServer.ContainedLanguage.Extensions;
+using Xunit;
+
+namespace Microsoft.VisualStudio.LanguageServer.ContainedLanguage
+{
+    public class UriExtensionsTest
```

Comment:
Same as the other `UriExtensionsTest` class, but since we have two separate extension classes I think we should test both separately as well